app/models/award.rb

```diff
@@ -12,4 +12,23 @@ def short_name
 
     award_name.gsub(regex, "")
   end
+
+  def needs_nominee_input?
+    type = award_type
+    if type == "directing" || type == "technical" || type == "acting"
+      return true
+    else
+      return false
+    end
+  end
+
+  def needs_role_input?
+    type = award_type
+    if type == "acting"
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.